File: client-app/src/Game/Models/field.ts
export class field {
  Creator: string | null = null;
  Height: number | null = null;
  Widht: number | null = null;

  public constructor(height: number, widht: number, login : string){
      this.Height = height;
      this.Widht = widht;
      this.Creator = login;
  }
}

File: Domain/Field.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Field
    {
        public string Id { get; set; }

        public int Height { get; set; }
        public int Widht { get; set; }


        public string AppUserId { get; set; }
        public AppUser AppUser { get; set; }


        public IList<Location> Locations { get; set; }
    }
}
